mender-client-acceptance-testing/Dockerfile

```diff
@@ -16,10 +16,26 @@ ENV LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8
 COPY requirements-py3.txt .
 RUN pip3 install -r requirements-py3.txt
```

Comment:
Also, this is dangerously getting out of sync of [meta-mender reqs file](https://github.com/mendersoftware/meta-mender/blob/master/tests/acceptance/requirements_py3.txt). I don't have a good suggestion on what to do, as dependabot will keep updating the file in meta-mender and not here (and generally speaking, we want the file there for users to run the tests easily)

Reply:
I would suggest we remove that one, and point people to this one instead.

And can we not make dependabot update this file too?

Reply:
@kacf Dependabot could update this one, but we cannot test the it (with current design) until it is already merged/deployed, which is too late. We have a similar problem for backend acceptance tests, see [QA-237](https://tracker.mender.io/browse/QA-237).

If we decide to solve this in a similar way, would mean that an update here shall run client acceptance tests (maybe is enough with only one config) before approving/merging/updating the base test image. I can create a task for it straight away or discuss it further on a QA meeting.

With regards to where to point the users to, I have no objections on pointing them here. But still we need to fix the above.

Reply:
here dependabot is blocked by the existing PRs (as the default max is 5 PRs from dependabot in a repo)... this could be set higher to get around the blockage...
if the dependencies are coupled to the meta-mender repo they should be in one place... is this image used outside of the context of anything related to meta-mender? if not I'd say it should live alongside the tests in the meta-mender repo...
if it is used outside of meta-mender, do the tests in meta-mender ever run without this image? if yes, we could just pull in the dependencies from there and if no, the dependencies could live in this repo...

Reply:
I know, but this is a problem even without dependabot. You mentioned running the acceptance tests from here, but how would we do that? We would have to change `mender-qa/.gitlab-ci.yml` to use a temporary image name. This sounds quite complicated to do from the integration-test-runner, so is it worth it? Not sure... I'm not against it, but I think the risk of breakage here is pretty low, and whoever merges this should immediately launch a mender-qa/master build.

Maybe we could set dependabot to a somewhat more infrequent schedule, if possible? It is not urgent at all to update these dependencies, and this way we would at least not have to deal with it very often.

Reply:
@mzedel @kacf I have a suggestion:

* Dependencies live in meta-mender
* This repo's Dockerfile does wget requirements from master + pip3 install (similar to what I have added in this PR)
* mender-qa script, regardless of the base image, does a pip3 update requirements -> This is the key part to actually be able to test updates on the requirements

Note that with the above we install dependencies from master, and there is an slight change of these to later break a test in dunfell/zeus. But that I think is really marginal risk...

Reply:
Agreed. And that risk is the same with the current setup anyway, so we are not increasing it. :-)

Reply:
FYI, this is now implemented.